CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project are documented in this file.
 
+## 0.0.11
```

Comment:
Can you please add the date here, see https://github.com/fluxcd/kustomize-controller/blob/main/CHANGELOG.md

Reply:
Done!